bosh/releases/pre_render_scripts/diego-cell/cflinuxfs3-rootfs-setup/patch_pre-start.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+target="/var/vcap/all-releases/jobs-src/cflinuxfs3/cflinuxfs3-rootfs-setup/templates/pre-start"
+
+# Use the ephemeral data directory for the rootfs
+patch --verbose "${target}" <<'EOT'
+@@ -3,8 +3,8 @@
+
+ CONF_DIR=/var/vcap/jobs/cflinuxfs3-rootfs-setup/config
+ ROOTFS_PACKAGE=/var/vcap/packages/cflinuxfs3
+-ROOTFS_DIR=$ROOTFS_PACKAGE/rootfs
+-ROOTFS_TAR=$ROOTFS_PACKAGE/rootfs.tar
++ROOTFS_DIR=/var/vcap/data/rep/cflinuxfs3/rootfs
```

Comment:
I think with the mechanism we have now in place to override environment variables through `quarks` properties, we don't need this, but rather set the `ROOTFS_PACKAGE` env var with the ops-file.